### ISSUE TITLE
Tthe ponded area should not be in the Plugin tables or dialogs for editing

### ISSUE THREAD

**Issue by FLO-2DJJ** [FLO-2DSoftware/qgis-flo-2d-plugin#1229]
From @FLO-2DNoemi email (03-20-2024):
The ponded area should not be in the Plugin tables or dialogs for editing, and should not be a grayed-out field.  The plugin should have that variable in the background and just write a zero to the INP for those tables where it is needed. Because it is not needed by the FLO-2D SD, this generates confusion for users. Let's resolve this matter definitively in Plugin 1.0.

**Comment by FLO-2DJJ** [comment]
- [ ] Careful!: Remove ponded area from _user_swmm_nodes_ and _user_swmm_storage_units_ tables.
- [ ] Remove ponded area from _Storage Unit Tab_ in **Select components from shapefile layer**.
- [ ] Remove ponded area control and column of table from _Storage Units dialog_.
- [ ] Write zero to ponded areas of .INP (I think it is already done).

**Comment by FLO-2DKaren** [comment]
@FLO-2DNoemi 
Noemi,

If someone had a storage unit that was off site, could they still need the ponded area feature?

**Comment by FLO-2DNoemi** [comment]
No, the storage area has its storage curve for the geometry of the storage
tank. This is a separate feature.

The ponded area is a fictitious storage of water, you can see this as a
fictitious water tank connected to the end of the vertical pipes, that
stores water when flow exceeds the RIM, this smooths the results when the
system is overwhelmed and controls PH oscillations. This is used by EPA
SWMM because they have a lumped model (not a distributed model), in other
words , their capacity to calculate the distribution of water on the
surface and the interaction between the surface and SD layers is very
limited and they need this simplification.

We need to eliminate any ponded area from the plugin, but we still need to
read/write them to the INP to maintain compatibility with the EPASWMM GUI.
In simple words, the ponded area smooths the PH oscillations and eliminates
jumps when the capacity is exceeded in the pipe. This is a rough
simplification of the physical interaction between the surface system and
the SD.

**Comment by FLO-2DKaren** [comment]
done
